src/components/inputs/Selector.tsx

```diff
@@ -0,0 +1,144 @@
+import { Avatar, Box, Stack, Typography } from '@mui/material'
+import { SecondaryBtn } from '../buttons/SecondaryBtn'
+import { StyledAutocomplete } from './Autocomplete'
```

Comment:
Let's avoid using relative imports :bow:

Reply:
addressed in ce20cc7d9007779f481a24b6ae555a8ac039d0bc commit